src/__init__.py

```diff
@@ -273,13 +277,17 @@ def generate_basic(self):
             
             # remove the item we're about to place from the pool so it isn't placed twice
             self.multiworld.itempool.remove(item_to_place)
+            
+        # Generate item_counts here so it could be acessed in after_generate_basic
+        if self.player not in self.item_counts:
+            real_pool = self.multiworld.get_items()
+            self.item_counts[self.player] = {i.name:real_pool.count(i) for i in real_pool if i.player == self.player}
 
         after_generate_basic(self, self.multiworld, self.player)
-
         # Uncomment these to generate a diagram of your manual.  Only works on 0.4.4+
 
         # from Utils import visualize_regions
-        # visualize_regions(self.multiworld.get_region("Menu", self.player), f"{self.game}.puml")
+        # visualize_regions(self.multiworld.get_region("Menu", self.player), f"{self.game}_{self.player}.puml")
```

Comment:
I'm not sure this is desirable

Reply:
It help when you gen for multiple users with different removed locations, otherwise it only generate one graph with the last player's options

Reply:
but if unwanted will remove, it was originally acidentally added since the apworld im using to test and program had that